### DISCUSSION TITLE
[FeatureRequest] Table column filter by values "check all" functionality

### DISCUSSION THREAD

**Discussion by memfiz** [grafana/grafana#38279] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

Column filter by values needs select all functionality. Similarly, it has a "Clear filter" when you want to clear all checked values.

**Why is this needed**:

When using big tables there is often a need to filter by values to select many entries. Then you use the filter you can find e.g. 100 of values and there is no possibility to select them all with one click, you need to check them one by one, that is not feasible with such a large number of entries.

As an example
[https://play.grafana.org/d/baqo21CGz/column-filter-select-all-request?orgId=1](https://play.grafana.org/d/baqo21CGz/column-filter-select-all-request?orgId=1)
If you would like to select values of _id column that contain "5ndM" there is no way to check them all with one click after the filter finds them.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by MalinSchei** [comment]
Hi there,
I'm wondering when this feature will be available in Grafana? The filter function is useless without it.

**Reply by bruric** [reply]
> Hi there, I'm wondering when this feature will be available in Grafana? The filter function is useless without it.

I'm also interested in this feature, let us know if anything moves!

**Reply by Fursel** [reply]
Hi, same here it would be great to have this implemented

**Comment by AntoniusFirst** [comment]
Hi guys and in particular Grafana team,

Could you provide any informaton about plans to implement this feature? It's very long awaited feature for us. Thank you!

**Comment by stefanLeo** [comment]
Any update on this topic? @pkolyvas ?

**Comment by ChinmoyPadhi** [comment]
Any more update on this topic? @pkolyvas? Eager to hear on this.

**Comment by skeagy** [comment]
Echoing others' feedback- "filter table columns" can't be considered to be supported without this. Extremely narrow use case covered by existing implementation. Default should be to select all rows that match the filter (e.g. see jQuery Datatables column filtering for standard user expectations).

**Comment by sesoko88** [comment]
Adding to the support for this request. It makes working with less structured data more functional.

**Comment by LennyVdB** [comment]
Hi there, any update on this feature please? As already mentioned, without this it's pretty useless.
Thanks!

**Comment by fabio-t** [comment]
This is still a very sorely needed feature, is there any update? :pray:

**Comment by ahuarte47** [comment]
Hi everyone, I am proposing this [MR](https://github.com/grafana/grafana/pull/79290) that implements two new buttons to easily select or unselect subsets of columns values. All comments are wellcomed

**Comment by natellium** [comment]
Hey there! We want to let you all know that we are moving to Github Issues for feature requests. You can read more about the move [here](https://github.com/grafana/grafana/discussions/73424).

I will be moving this discussion, and you can follow it [here](https://github.com/grafana/grafana/issues/79371). Sorry for the inconvenience, and thanks for your contributions!

cc @grafana/grafana-bi-squad

**Reply by ahuarte47** [reply]
Thank you very much @natellium
